modules/network-observability-loki-install.adoc

```diff
@@ -5,7 +5,7 @@
 :_content-type: PROCEDURE
 [id="network-observability-loki-installation_{context}"]
 = Installing the Loki Operator
-It is recommended to install link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7], This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
+It is recommended to install link:https://catalog.redhat.com/software/containers/openshift-logging/loki-rhel8-operator/622b46bcae289285d6fcda39[Loki Operator version 5.7+]. This version provides the ability to create a LokiStack instance using the `openshift-network` tenant configuration mode. It also provides fully automatic, in-cluster authentication and authorization support for Network Observability.
```

Comment:
I know you didn't change that, but I don't think you need the comma there because `fully automatic` is modifying `in cluster authentication...`.

Reply:
This is a rebase nightmare waiting to happen because I got other peer review comment on this intro in #62901. I'm going to combine the two feedbacks and make them consistent in both PRs